### PR TITLE
Corrected OS terminology

### PR DESCRIPTION
**Pull request by chrishunter-konsentus**

### CODE REVIEW

File: scripts/helpers [hunk 1]
```diff
@@ -553,3 +553,16 @@ npm_install() {
 		echo "$(color red)[✗]$(color reset)"
 	fi
 }
+
+# Check to determine which OS am I running on for more informative messages
+
+which_os() {
+    unameOut="$(uname -s)"
```
Comment (davidjdixon) [inline scripts/helpers:560]: use `local x` to ensure the var isn't exposed globally

**Review by davidjdixon — CHANGES_REQUESTED** [review]
at the moment, this is missing the actual use of `which_os` so this would case empty strings to be printed

File: scripts/helpers (hunk 1 above)
Comment (chrishunter-konsentus) [reply scripts/helpers]: Good thought. Added.

Comment (chrishunter-konsentus) [reply scripts/helpers]: Added missing `which_os`. This will teach me to split up a commit.

File: scripts/clean [hunk 2]
```diff
@@ -38,7 +38,7 @@ done
 
 sudo -k
 echo "$(color red)Warning: this will erase your Homebrew and ZSH configuration$(color reset)"
-echo -n "Enter Your MacOS "
```
Comment (davidjdixon) [inline scripts/clean:41]: still missing use of which_os for the clean script 🙁

**Review by davidjdixon — CHANGES_REQUESTED** [review]
... might be easier to push all changes instead of trying to break them apart as its leaving portions missing.

File: scripts/clean (hunk 2 above)
Comment (chrishunter-konsentus) [reply scripts/clean:41]: @davidjdixon I think you commented on the old code rather than the change

image

**Comment by chrishunter-konsentus** [comment]
Closing for the moment

**Pull request closed without merging**